grammar-tools/MySQLParser.g4

```diff
@@ -153,12 +153,12 @@ alterStatement:
         | alterLogfileGroup
         | alterServer
         // ALTER USER is part of the user management rule.
-        | alterInstance /* @FIX: Add support for "ALTER INSTANCE ..." statement. */
+        | alterInstance /* @CHANGED: Add support for "ALTER INSTANCE ..." statement. */
```

Comment:
Perfect, this `@CHANGED` annotation is super clear

Reply:
Maybe past tense for the comment? So "Added support"?

Reply:
Good idea, fixed.